modules/quality_control/jsx/qualityControlIndex.js

```diff
@@ -0,0 +1,225 @@
+import React, {Component} from 'react';
+import Loader from 'Loader';
+import {Tabs, TabPane} from 'Tabs';
+import FilterableDataTable from 'FilterableDataTable';
+import PropTypes from 'prop-types';
+
+class QualityControlIndex extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ImgData: {},
+      BehavioralData: {},
+      isLoadedImg: false,
+      isLoadedBehavioral: false,
+      imgFilter: {},
+      behavioralFilter: {},
+    };
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   *
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {object} row - row content indexed by column
+   *
+   * @return {*} a formated table cell for a given column
+   */
+
+  formatColumn(column, cell, row) {
+      let result = <td>{cell}</td>;
+      switch (column) {
+      case 'Scan Done in MRI PF':
+        if (cell == 'Yes') {
+          let mpfURL = loris.BaseURL + '/mri_parameter_form/?commentID=' +
```

Comment:
@llevitis The new url as per major should be `loris.BaseURL + 'instruments/mri_parameter_form/?commentID=' +`

Reply:
@sruthymathew123 That's a good point. I think this could be deduced using the value in the visit column for IBIS, but I'm not sure what the naming is like for other projects that have multiple MPFs. @ridz1208, thoughts?